searchcore/src/vespa/searchcore/proton/server/feedhandler.h

```diff
@@ -96,6 +96,7 @@ class FeedHandler: private search::transactionlog::client::Callback,
     std::mutex                             _syncLock;
     SerialNum                              _syncedSerialNum; 
     bool                                   _allowSync; // Sanity check
+    std::atomic<uint64_t>                  _heart_beat_time;
 
```

Comment:
I do see that we have test for atomic time.
static_assert(std::atomic<steady_time>::is_always_lock_free, "std::atomic<steady_time> should be atomic");
Then I suggest that you use std::atomic<steady_time>. Then you avoid all the casting.